app/Http/Controllers/User/BookmarksController.php

```diff
@@ -222,7 +225,9 @@ public function update(Request $request, $user_id, $bookmark_id)
         if (!$bookmark) {
             return $this->setStatusCode(404)->replyWithError('Bookmark not found');
         }
-        $bookmark->fill($request->all());
+        $current_bookmark = $request->all();
+        $current_bookmark['verse_sequence'] = $request->verse_sequence ?? (int) $request->verse_start;
```

Comment:
same concern as above - casting string to int when string is "3a"

Reply:
@bradflood  when a string is cast to an integer, any non-numeric characters will be removed, leaving only the numerical value.

E.g. "2aedA" => 2